src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -132,16 +133,24 @@ async void InitializeAsync()
             VirtualFolderPath = string.Empty;
             try
             {
-                if (viewModel.Link != null && !string.IsNullOrEmpty(viewModel.CurrentPackageName))
+                //if this node is from a package then we set the virtual host path to the packages docs directory.
+                if (viewModel.Link != null && !string.IsNullOrEmpty(viewModel.CurrentPackageName) && viewModel.IsOwnedByPackage)
                 {
```

Comment:
this logic was backwards for both shared .dyns and shared images - if this is a package node we should NOT set the virtual host name to map to the shared folder, we should set it to the package's doc folder.

If the node is builtin we should use the shared folders.

Reply:
FYI @Amoursol @QilongTang